releasenotes/notes/bump-bifrost-xena-c0c65bd823ed15fa.yaml

```diff
@@ -0,0 +1,6 @@
+---
+fixes:
+  - |
+    Rebuild and bump the Bifrost container for Xena to include fix for
+    `Error while running update_to_latest_versions: ''BIOSSetting''
+    object has no attribute` during `Ironic database migrations on upgrade`
```

Comment:
Newline EOF

Reply:
Done

Reply:
> Done

are you sure? https://github.com/stackhpc/stackhpc-kayobe-config/compare/8546ab49632989292faea475a82a38d4e42423a9..49cf122387fc55d893a6ee2d68e096088aa68c81

Reply:
Apologies. I had --amended the commit without staging the changes.